src/plugins/home/server/services/sample_data/data_sets/util.test.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getSavedObjectsWithDataSource } from './util';
+import { SavedObject, updateDataSourceNameInVegaSpec } from '../../../../../../core/server';
+
+describe('getSavedObjectsWithDataSource()', () => {
+  const getVisualizationSavedObjects = (): Array<SavedObject<any>> => {
```

Comment:
nonblocking: you can move this large json object to a `data.json` file and then load it with one line

```
import data from './data.json'

```